client/src/assets/scss/_variables.scss

```diff
@@ -24,7 +24,7 @@ $color-lighter-blue:        #E0E7FF;
 $color-white:               #FFFFFF;
 $color-svg-grid:            #8097B1;
 $color-orange-dark:         #F7931E;
-$color-blue:                #4C06FF;
```

Comment:
Just want to make sure, are those above the only uses of `$color-blue`?

Reply:
@nedSaf 

Yes:

![Selection_986](https://user-images.githubusercontent.com/17882245/181229008-04db19e1-016b-4382-895f-3dd6cb784dfa.png)